### PR TITLE
add a command to launch a simple http server on python 3.x

### LINKED ISSUE #738: Support python 3.x to start Simple HTTP Server
## Overview
**TL;DR**
<One to two sentence description of the issue you are encountering or trying to solve.>

In the README.md, the procedure to create git and github cheat sheet. is well described.
There is a command to launch the simple HTTP server on python 2.x that does not 
work on python 3.x.

#### Screenshots
<If bug fix, please upload images of the error to better showcase the problem.>

#### Reasoning
<If new idea or direction, please create a checklist of proposed ideas or next steps. [Example](https://github.com/github/training-kit/issues/415).>

### How to Contribute
- [ ] Comment on this issue if you'd like to work/collaborate on it!
- [ ] Fork the [repository](https://github.com/github/training-kit).
- [ ] Use the [GitHub Flow](https://guides.github.com/introduction/flow/) to make changes to your fork. [This](https://services.github.com/on-demand/intro-to-github/) is a refresher course if you're unsure about how to make a change on GitHub.
- [ ] Push your changes to your repository.
- [v ] Submit a Pull Request
  - Base Dropdown: github/training-kit
  - Compare Dropdown: Your fork

#### Testing Locally
If you'd like to make and test changes locally (and see how they would look if merged), do the following:
- In your command line:
  - [Install Git](https://git-scm.com/)
  - [Install Ruby](https://www.ruby-lang.org/en/documentation/installation/)
  - [ ] Run `script/setup`
  - [ ] Run `script/server`.
    - When successful, the script will initiate a local server at `http://127.0.0.1:4000/`.

### Questions?
- Leave a comment on this issue! Make sure to use @ mentions if you want a specific person's attention!

### PR DESCRIPTION
**Pull request by geunkim**
## Overview
**TL;DR**
<One to two sentence description of the issue you are encountering or trying to solve.>

add a command to launch a simple http server on python 3.x

<Link to related issue. Type `closes #RELATEDISSUENUMBER` to establish a link.>
closes #738
### Questions
<If relevant, write a list of questions that you would like to discuss related to the changes that you have made.>

### Next Steps
<If incomplete, create a task list of items that are still being worked on within the Pull Request.>

### Review
<If complete, or ready for :eyes:, use @mentions for quick questions, specific feedback, and progress updates.>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ If you'd like to have a copy of the files to be served from a web server inside
 2. To test this looks okay, create some folders `mkdir -p test_site/kit`.
 3. Untar the release, `tar -xzf release-XXXXXXX.tgz -C test_site/kit`.
 4. Switch into the test_site directory, `cd test_site`.
-5. View the site with `python -m SimpleHTTPServer`. _Note: Some servers are more advanced than others and can handle redirects, smart recognition of `.html` files, etc_
+5. View the site with `python -m SimpleHTTPServer` for python version 2.x `python -m http.server` for python version 3.x. _Note: Some servers are more advanced than others and can handle redirects, smart recognition of `.html` files, etc_
```
Comment (brianamarie) [inline README.md]:
```suggestion
5. View the site:
    - For python version 2.x, run: `python -m SimpleHTTPServer`
    - For python version 3.x, run: `python -m http.server`
    - _Note: Some servers are more advanced than others and can handle redirects, smart recognition of `.html` files, etc_
```

**Comment by geunkim** [comment]
Hi @brianamarie, I accept your suggestion. I am newbie for GitHub. 
Please let me know when I make a mistake. 
I would like to report a bug report later and modify the korean cheatsheet.
Thanks.

**Comment by geunkim** [comment]
Hi @brianamarie,

**Comment by geunkim** [comment]
Hi @brianamarie, I agree with your comment.
Please let me know what should I do.
